tests/Integration/RouterTest.php

```diff
@@ -14,15 +14,19 @@ class RouterTest extends \Tests\TestCase
     public function it_returns_correct_urls_for_host()
     {
         putenv('API_HOST'); // omitting value removes the env var
+        putenv('API_SCHEME');
         $this->assertEquals('https://api.ethicaljobs.com.au/jobs', Router::getRouteUrl('jobs'));
```

Comment:
I think we need testa with API_HOST empty and API_SCHEME set and the reverse: API_HOST empty and API_SCHEME set.

Reply:
Good point. Added

Reply:
Great!